docs/index.html

```diff
@@ -523,10 +587,31 @@
       {
         quadrant: 0,
         ring: 2,
-        label: "TypeScript",
+        label: "Kotlin Script",
+        active: true,
+        moved: 0
+      },
+      {
+        quadrant: 0,
+        ring: 2,
+        label: "Kotlin Multiplatform",
+        active: true,
+        moved: 0
+      },
+      {
+        quadrant: 0,
+        ring: 2,
+        label: "Go",
```

Comment:
can we add a article for this describing for what it's used (AFAIK devops/CI things and the fastest-paths experiment)

Reply:
Added the link and created a preliminary page. Feel free to adapt.